js/script.js

```diff
@@ -244,3 +244,45 @@ function meadiaQueryClose() {
 
 checkMeadiaQuery();
 meadiaQueryClose();
+
+// Validation FORMS //
+
+const form = document.querySelector('form');
+const firstName = document.querySelector('#firstname');
+const lastName = document.querySelector('#lastname');
+const email = document.querySelector('#email');
+const textArea = document.querySelector('#text');
+const errorFName = document.querySelector('#sm-1');
+const errorLName = document.querySelector('#sm-2');
+const errorEmail = document.querySelector('#sm-3');
+const errorText = document.querySelector('#sm-4');
+
+form.addEventListener('submit', (element) => {
+  if (firstName.value === '' || firstName.value == null) {
+    element.preventDefault();
+    errorFName.innerText = 'First name is required';
+  } else {
+    errorFName.innerText = '';
+  }
+  if (lastName.value === '' || lastName.value == null) {
+    element.preventDefault();
+    errorLName.innerText = 'Last name is required';
+  } else {
+    errorLName.innerText = '';
+  }
+  if (email.value === '' || email.value == null) {
+    element.preventDefault();
+    errorEmail.innerText = 'Email is required';
+  } else if (email.value.toLowerCase() !== email.value) {
+    element.preventDefault();
+    errorEmail.innerText = 'Please fill in your email address in lower case';
+  } else {
+    errorEmail.innerText = '';
+  }
+  if (textArea.value === '' || email.value == null) {
+    element.preventDefault();
+    errorText.innerText = 'Text is required';
+  } else {
+    errorText.innerText = '';
+  }
+});
```

Comment:
- [ ] The logic you wrote here for validating the form doesn't seem to be working. At the moment, the form gets submitted even if it's only one field that has a value or the email has an uppercase letter. You are required to validate the email field not to have uppercase text but all lowercase text before the form is submitted. Kindly work on that in a pair programming way with your partner 👍 
- [ ] Also, to prevent the form from being submitted without filling all fields, kindly add the required attribute to all your input fields as this will help with basic form validation for the input fields. 👍 
- [ ] The screenshot below is getting submitted when I hit the submit button. Please fix that so that email with uppercase text does not allow the form to be submitted 👍

![image](https://user-images.githubusercontent.com/50941074/203937986-f34f89f4-f5b5-4421-8465-89889f0ad9d3.png)

Reply:
Hi @AmaduKamara, I just realized that I used wrong `id` name in the script.js file that's why it wasn't working. Now it is fixed. Would you consider checking the code again? 

Thank you